### PR TITLE
aio: Update getting started deployment instructions 

### PR DESCRIPTION
**Pull request by StephenFluin**
NOTE!
This is intended to be merged after https://github.com/angular/angular/pull/29651 is updated and merged.

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit

## PR Type
What kind of change does this PR introduce?

- [x] Documentation content changes

### CODE REVIEW

File: aio/content/start/deployment.md [hunk 1]
```diff
@@ -13,20 +13,18 @@ Whether you came here directly from [Your First App](start "Getting Started: You
...
-StackBlitz allows you to publish your Angular app directly to Firebase from your project. The steps below outline how to deploy it quickly without setting up your own hosting environment.
+StackBlitz projects are public by default, allowing you to share your Angular app via the project URL. Keep in mind that this is a great way to share ideas and prototypes, but it is not intended for production hosting.
 
-1. In your StackBlitz project, in the left menu bar, click the `Firebase` icon.
-1. If you don’t have a `Firebase` account, visit the [Firebase](https://firebase.google.com/ "Firebase web site") to sign up for a free hosting account.
-1. Click the `Sign into Google` button and follow the prompts to give `StackBlitz` access your `Firebase` projects
-1. Select the `project` where you wish to deploy your application.
-1. Click the `Deploy` button to deploy your application.
-1. After the deployment completes, click the `Open live site` link to view your app live.
+1. In your StackBlitz project, make sure you have forked or saved your project
```
Comment (brandonroberts) [inline aio/content/start/deployment.md]: Add periods to the end of each sentence.

File: aio/content/start/deployment.md [hunk 2]
```diff
@@ -13,20 +13,18 @@ Whether you came here directly from [Your First App](start "Getting Started: You
...
+1. In the preview pane, you should see a URL that looks like `https://<Project ID>.stackblitz.io`
+1. Share this URL with a friend of colleague
```
Comment (brandonroberts) [inline aio/content/start/deployment.md]:
```suggestion
1. Share this URL with a friend or colleague.
```

**Review by brandonroberts — DISMISSED** [review]

**Review by IgorMinar — APPROVED** [review]

**Comment by brandonroberts** [comment]
Caretaker note: Codefresh build failure is unrelated.

**Review by Splaktar — APPROVED** [review]
LGTM.

I posted some details about how to improve this and include a production deployment after talking to the Stackblitz team at ng-conf. You can find [those details in #docs on Slack](https://angular-team.slack.com/archives/C8MK7DQJX/p1557169500007300).

**Pull request closed without merging**